scrapy/downloadermiddlewares/uriuserinfo.py

```diff
@@ -0,0 +1,49 @@
+from urllib.parse import unquote, urlunparse
+
+from scrapy.utils.httpobj import urlparse_cached
+
+
+class UriUserinfoMiddleware(object):
```

Comment:
:warning: _Nitpick alert_ :warning:
Should "info" be capitalized to "Info"? The [corresponding RFC](https://tools.ietf.org/html/rfc3986#section-3.2.1) says "userinfo", all lowercase, but if we're using camel case I think it might make sense to make it "UserInfo" like in [this example](https://docs.microsoft.com/en-us/dotnet/api/system.uri.userinfo?view=netframework-4.8)